src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,94 @@
+package duke;
+
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Parser {
+
+    /**
+     * Getting required info from user input
+     *
+     * @param command User input
+     * @return String list to store required info
+     * @throws DukeException
+     */
+    public static String[] parse(String command) throws DukeException {
```

Comment:
perhaps you could consider using a switch statement instead of else-ifs? since it is easier to read through a switch statement and debug any possible errors compared to else-ifs.